xbmcvfs.py

```diff
@@ -0,0 +1,176 @@
+# SPDX-License-Identifier: GPL-3.0
+
+import os
+from shutil import copyfile
+
+
+def File(path, flags='r'):  # NOSONAR
+    return open(path, flags, encoding='utf-8')
```

Comment:
Please use `io.open()` to improve cross platform and cross Python version compatibility.  

I also don't know if this will work, because a Python file object is returned, not a Kodi File object. The biggest difference is the missing `readBytes`. But for now, let's do it like this.

Reply:
I've written out the functions for the `File` class. I did notice that it's also possible to use it with the `with` syntax from Kodi 19:

```
with xbmcvfs.File(file, 'w') as f:
  result = f.write(buffer)
```

I need to look into this, but I think I need to implement a `contextmanager` for this, altough I'm not sure if this also works with classes.

Reply:
This was easy :)